__tests__/Port.test.js

```diff
@@ -0,0 +1,39 @@
+const Port = require("../src/Port");
+
+describe("Port", () => {
+  describe("ports and ships", () => {
+    let port;
+    let ship;
+    let titanic;
+    let queenMary;
+
+    beforeEach(() => {
+      port = new Port("name");
+      ship = jest.fn();
+      titanic = jest.fn();
+      queenMary = jest.fn();
+    });
+
+    it("can be instantiated", () => {
+      expect(new Port()).toBeInstanceOf(Object);
```

Comment:
instead of calling `new Port()` here, you could reference the `port` you have already defined i.e.
```
expect(port).toBeInstanceOf(Object);
```